authentication/views.py

```diff
@@ -41,20 +41,21 @@ def get(self, request):
 
 class LoginView(APIView):
     def post(self, request):
-        email = request.data.get('email')
-        password = request.data.get('password')
-        user = authenticate(email=email, password=password)
+        if request.method == 'POST':
```

Comment:
Using this control flow statement here is unnecessary, because DRF's APIView matches methods with corresponding HTTP methods by their names - .post(), .get(), etc.